Trunk.toml

```diff
@@ -9,6 +9,8 @@ release = false
 dist = "dist"
 # The public URL from which assets are to be served.
 public_url = "/"
+# Do not create implicit rust entry if it does not exist in HTML file
+no_implicit_rust = false
```

Comment:
I think it would be this attribute was named `implict_rust`. The double negation here makes it comparatively hard to understand

Reply:
You are right, but I do not know how to set default value to `true` in Serde, except that by creating function that returns true.